common/vpcclient/models/subnet.go

```diff
@@ -0,0 +1,50 @@
+/*******************************************************************************
+ * IBM Confidential
+ * OCO Source Materials
+ * IBM Cloud Kubernetes Service, 5737-D43
+ * (C) Copyright IBM Corp. 2023 All Rights Reserved.
+ * The source code for this program is not published or otherwise divested of
+ * its trade secrets, irrespective of what has been deposited with
+ * the U.S. Copyright Office.
+ ******************************************************************************/
+
+// Package models ...
+package models
+
+import (
+	"github.com/IBM/ibmcloud-volume-interface/lib/provider"
+)
+
+// Subnet ...
+type Subnet struct {
+	CRN           string         `json:"crn,omitempty"`
+	Href          string         `json:"href,omitempty"`
+	ID            string         `json:"id,omitempty"`
```

Comment:
is making ID as optional making sense or do we need to create two seperate struct one if for required and another would be for response

Reply:
Next PR will be analysing this comment https://github.ibm.com/alchemy-containers/armada-storage/issues/4583